docroot/themes/humsci/humsci_basic/src/scss/components/_postcard.scss

```diff
@@ -0,0 +1,107 @@
+.ptype-hs-postcard {
+  .hb-card {
```

Comment:
can you add a little comment here about how these are all override styles to get the postcard paragraph type working?